### ISSUE TITLE
bookmark loss upon renaming parent folder

### ISSUE THREAD

**Issue by j-lakeman** [PaulWoitaschek/Voice#2636]
### Overview

When renaming/moving the parent folder where the audiobooks are in, all bookmarks seem to be lost that were previously set for an audiobook.
I'm assuming it must've sth to do with saving absolute path info rather than relative one.

### How to reproduce

1. rename parent folder of audiobooks
2. bookmarks are gone

### Version

8.2.3

### Android Version

14

### Device Model

Google Pixel 5

**Comment by luzpaz** [comment]
Is this still an issue? I'm glad i looked this up because I was tempted to rename but was worried bookmarks would get lost.

**Comment by luzpaz** [comment]
There is also an open discussion: 
* https://github.com/PaulWoitaschek/Voice/discussions/3221

**Comment by luzpaz** [comment]
This is problematic for me. I am using Voice to take meticulous timecode bookmarked notes. And If I move the directory the audio file is in, these bookmarks are lost!

**Comment by luzpaz** [comment]
@PaulWoitaschek any chance you could consider this ?

**Comment by luzpaz** [comment]
Hi @phreeza, any chance you can take a look at this issue related to 'Voice' ? Thanks in advance! 

Edit: note this impacts #3221 as well

**Comment by luzpaz** [comment]
#3221 is also impacted by the same problem

**Comment by phreeza** [comment]
Not sure why I'm being tagged, I am not a maintainer on this project.

**Comment by luzpaz** [comment]
Yea, sorry, in my eagerness to get this issue looked at there was no context.. Would you have any interest in fixing this issue ? It makes the feature of bookmarks very fragile to use with the app. Maybe this issue impacts you enough as well for you to consider investigating ?

**Comment by j-lakeman** [comment]
@luzpaz yep still an issue for me. Interestingly if you rename the parent folder back to the original title, the bookmarks reappear. That suggests the path names are not updated properly in a database IMHO.

**Comment by luzpaz** [comment]
Where is the database in the code ?

**Comment by j-lakeman** [comment]
Idk m8 it's more of an impression I get as a user 🤷‍♂️

**Comment by luzpaz** [comment]
@PaulWoitaschek any chance you can take a look at this ?

**Comment by luzpaz** [comment]
@PaulWoitaschek is there a workaround for this ?

**Comment by luzpaz** [comment]
CC @DevOelgaard any chance you have an interest in this bug ? Does it impact you at all ?

Edit: 
And perhaps https://github.com/PaulWoitaschek/Voice/discussions/2549?notification_referrer_id=NT_kwHOAD8s19oAJlJlcG9zaXRvcnk7MjI1NTgzODc7RGlzY3Vzc2lvbjs3MTk5OTY3&notifications_query=repo%3APaulWoitaschek%2FVoice#discussioncomment-16692082 ?

**Comment by DevOelgaard** [comment]
Not quite sorry it doesn't really affect how I use Voice and I haven't experienced it.

If I have some free time after vacation I might look at it anyway, but no promises :)

**Comment by DevOelgaard** [comment]
actually @luzpaz this looks intriguing, I'll give it a look when time allows :)

**Comment by luzpaz** [comment]
@DevOelgaard any news ? Thanks for being interested.
